### ISSUE TITLE
Composer fails with generic "your data was bad" error message if alt text is longer than 140 characters

### ISSUE THREAD

**Issue by BenLubar** [pixelfed/pixelfed#1909]
This limit is not exposed to the user in any way, and the error message is dropped from the packet.

Here's a screenshot from Wireshark:

![image](https://user-images.githubusercontent.com/4257305/71651532-f9e5b280-2ce3-11ea-9a39-428ebea034de.png)

**Comment by dansup** [comment]
@BenLubar We have deprecated the old ComposeUI, the new version has a character count. Update to v0.10.7 😎
